public/app/plugins/datasource/prometheus/querybuilder/components/MetricsLabelsSection.tsx

```diff
@@ -60,7 +60,6 @@ export function MetricsLabelsSection({
   const onGetLabelNames = async (forLabel: Partial<QueryBuilderLabelFilter>): Promise<SelectableValue[]> => {
     // If no metric we need to use a different method
     if (!query.metric) {
-      // FIXME pass timeRange to fetchLabels method
```

Comment:
🤦 So now we don't need to pass the timeRange. Because it was done in `PromQueryBuilderContainer`. See below.